File: mcachedb/python/delete/mcachedb_delete.py
#! /usr/bin/python3
# -*- coding: utf-8 -*-
#
#	mcachedb_delete.py
#
#					Feb/02/2015
#
import	sys
#
import memcache
#
sys.path.append ('/var/www/data_base/common/python_common')
from mcache_manipulate import mcache_delete_proc
# ------------------------------------------------------------
print ("*** 開始 ***")
#
key_in = sys.argv[1]
print ("%s" % key_in)
#
mc = memcache.Client(['ubuntu_1504:21201'])
#
mcache_delete_proc (mc,key_in)
#
print ("*** 終了 ***")
#
# ------------------------------------------------------------

File: mcachedb/ruby/delete/mcachedb_delete.rb
#! /usr/bin/ruby
# -*- coding: utf-8 -*-
#
#	mcachedb_delete.rb
#
#						Mar/22/2012
#
require 'memcache'
require 'json'
#
load '/var/www/data_base/common/ruby_common/mcache_manipulate.rb'
#
# ------------------------------------------------------------
puts "*** 開始 ***"
#
id_in = ARGV[0]
#
puts id_in
#
cache = MemCache.new 'localhost:21201'
#
#
cache.delete(id_in)
#
keys=["t1521","t1522","t1523","t1524",
	"t1525","t1526","t1527","t1528",
	"t1529","t1530","t1531","t1532"]
keys.each {|key|
	display_record_proc(cache,key)
	}
#
#
puts "*** 終了 ***"
# ------------------------------------------------------------

